coreos-ostree-importer/coreos_ostree_importer.py

```diff
@@ -148,6 +173,18 @@ def process(self, message: fedora_messaging.api.Message):
             )
 
 
+def assert_dirs_group_writable(repo: str):
```

Comment:
Minor: IMO it would be cleaner if this function just took the path to the repo to check to make it more self-contained. We already have it handy in both callsites.

Reply:
ok will do